composer.json

```diff
@@ -39,5 +39,11 @@
         "branch-alias": {
             "dev-master": "1.2-dev"
         }
-    }
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/danrot/jackalope.git"
```

Comment:
this can go away when https://github.com/jackalope/jackalope/pull/286 is merged, right?